server/services/yelp.js

```
@@ -11,7 +11,7 @@ export async function searchPlaces(params) {
   try {
     const { data } = await authedAxios.get('/businesses/search', { params });
     return data.businesses;
-  } catch (e) {
-    console.log('search failed!', e);
+  } catch (e) {  
+    return [];
```

we dont want silently failed request right?